sass/components/_asset-card.scss

```diff
@@ -44,6 +44,7 @@ $asset-card-padding: 0.4rem;
     font-size: 1.2rem;
     background-color: var(--asset-card-title-bg-color);
     color: var(--asset-card-title-color);
+    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
```

Comment:
In general, colors shouldn't be used directly, define a `--asset-cart-title-text-shadow-color` (verbose… 🤮) in `_vars.scss` in its corresponding section. The reason is that this will allow to change the shadow color for light mode.

@rparrett FYI, in case you see a similar issue in other PRs.

Reply:
Anyway, no need to do the change unless we agree that we want the text shadow.